### ISSUE TITLE
1.3.60-eap-76 leads to EXC_BAD_ACCESS (code=1, address=0x8) when extends abstract class

### ISSUE THREAD

**Issue by subprogram** [JetBrains/kotlin-native#3503]
The bug is introduced in 1.3.60-eap-76. Version 1.3.50 works fine. I managed to make a reproducer.

File common/Sample.kt:
```
fun test(foo: IFoo) = foo.f()

interface IFoo {
    fun f()
}
```

File iosMain/Sample.kt:
```
abstract class IosFooBase : IFoo {
    override fun f() = bar()

    abstract fun bar()
}
```

File ViewController.swift:
```
class ViewController: UIViewController {
    override func viewDidLoad() {
        super.viewDidLoad()

        SampleKt.test(foo: IosFoo())    //EXC_BAD_ACCESS (code=1, address=0x8)
    }
}


class IosFoo : IosFooBase {
    override func bar() {}
}
```

I've found a workaround - if I replace `abstract class` by `open class`, the bug is disappeared:
```
open class IosFooBase : IFoo {
    override fun f() = bar()

    open fun bar(): Unit = TODO("Should be implemented")
}
```

**Comment by SvyatoslavScherbina** [comment]
Thanks, confirmed.

**Comment by homuroll** [comment]
Fixed in master and should be fixed in the next EAP of 1.3.60.
